components/style/properties/helpers.mako.rs

```diff
@@ -33,6 +33,9 @@
         % if initial_specified_value:
         #[inline] pub fn get_initial_specified_value() -> SpecifiedValue { ${initial_specified_value} }
         % endif
+        % if none_specified_value:
+        #[inline] pub fn get_none_specified_value() -> SpecifiedValue { ${none_specified_value} }
```

Comment:
This looks like it's only specific to the `font-variant-ligatures` property. Can we make it so that callers of `get_none_specified_value()` calls `specified::FontVariantLigatures::none()` directly?

Reply:
Yeah, indeed. Let's change the callers of this to be:

```rust
FontVariantLigatures::none()
```

And revert the changes to this file.

Reply:
@KiChjang @emilio Thanks for suggestion. I'll try to use `specified::FontVariantLigatures::none()` directly from the caller! It looks better for me!